detection/notebooks/00_webcam.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p><code>get_bounding_boxes</code> now returns a list of AnnotationBbox</p><p><code>plot_boxes</code> also takes in different params, AnnotationBbox</p><p> </p><p>This <code>process</code> function does prediction and plotting together. I'm not sure if its the best, but should we split them up? Seems like they are two distinct steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think it's fine for this notebook, since it's just to demo the basic steps and usage.  What do you think?</p>

Reply:
<p>It makes sense.</p><p> </p>